crates/api_common/src/utils.rs

```diff
@@ -537,23 +537,15 @@ pub async fn get_url_blocklist(context: &LemmyContext) -> LemmyResult<RegexSet>
         let urls = LocalSiteUrlBlocklist::get_all(&mut context.pool()).await?;
 
         let regexes = urls.iter().map(|url| {
-          let url = &url.url;
-          let parsed = Url::parse(url).expect("Coundln't parse URL.");
-          if url.ends_with('/') {
-            format!(
-              "({}://)?{}{}?",
-              parsed.scheme(),
-              escape(parsed.domain().expect("No domain.")),
-              escape(parsed.path())
-            )
-          } else {
-            format!(
-              "({}://)?{}{}",
-              parsed.scheme(),
-              escape(parsed.domain().expect("No domain.")),
-              escape(parsed.path())
-            )
-          }
+          // The scheme is removed in the saving,
+          // so fake it here to build the url.
+          let url = &format!("https://{}", url.url);
```

Comment:
I'm debating whether we need to do this at all. The urls are already validated on saving, and it seems pointless to re-parse them for the regex.

Reply:
Yeah I'm just gonna do it.